content/webapp/pages/exhibition.js

```diff
@@ -192,6 +197,7 @@ export class ExhibitionPage extends Component<Props, State> {
         icon: 'a11y'
       },
       {
+        id: null,
```

Comment:
Ah, scrap what I said above, we use this too many times manually for it to make sense...